src/main/java/it/smartcommunitylab/aac/webauthn/model/WebAuthnUserIdentity.java

```diff
@@ -88,16 +98,14 @@ public void setUserId(String userId) {
 
     @Override
     public void eraseCredentials() {
-        this.account.setCredentials(null);
+
+        List<WebAuthnCredential> credentials = webAuthnCredentialsRepository.findByParentAccountId(account.getId());
+        webAuthnCredentialsRepository.deleteAll(credentials);
```

Comment:
this is incorrect. The purpose of eraseCredentials is not to remove entities from repositories, but only to clear **this** object from credentials data to make sure we can export/print/expose this DTO. Any operation should modify **only** this single instance.

Reply:
Solved in 4c1cf14bfbc03a2d50dfb11eaf82415db58ea12c.
Now the `eraseCredentials` method does nothing, since we do not have any type of 'credential' in the WebAuthnUserAccount.